.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-gemfiles/Gemfile-1.9.lock
\ No newline at end of file
+gemfiles/Gemfile-1.9.lock
+.DS_Store
```

Comment:
I added this because OSX Finder does this with annoying regularity.

I'll put it in a separate PR.